### ISSUE TITLE
nipy fails to install using `poetry add` with mac os

### ISSUE THREAD

**Issue by jiavila** [nipy/nipy#482]
Hi!

I tried using `poetry add nipy` on my mac but it fails to install. I keep getting the following

**Comment by matthew-brett** [comment]
I don't know poetry I'm afraid - but

* What architecture is your Mac running on?  Intel or M1?
* Can you install Numpy in a previous step?

**Comment by jiavila** [comment]
2.6 GHz 6-Core Intel Core i7

Yes, numpy can be installed separately but, regardless, nipy always tries to build numpy when trying to install with poetry

**Comment by matthew-brett** [comment]
Yes, I've seen a problem like this now.   For the continuous integration, I worked round it thus: 

https://github.com/MacPython/nipy-wheels/blob/master/config.sh#L4

In summary, I install OpenBLAS with homebrew, and then configure Numpy to use it instead of Accelerate.   I don't know how else to fix this, I'm afraid.

**Comment by effigies** [comment]
@jiavila Is this resolved in the latest release?
